lib/elaborate/elaborate-use-clauses.ts

```diff
@@ -45,6 +48,14 @@ export function elaborateUseClauses(file: OFile, projectParser: ProjectParser, v
             if (implementsIHasPackageInstantiations(iterator)) {
               pkgInstantations.push(...iterator.packageInstantiations);
             }
+            if (implementsIHasGenerics(iterator)) {
+              for (const generic of iterator.generics) {
+                if (generic instanceof OInterfacePackage) {
+                  pkgInstantations.push(generic);
+                }
+              }
+
+            }
```

Comment:
This doesn't really belong into `elaborateUseClauses` but should be ok...

Reply:
Its part of the mechanism to find the package for the use-clause. I think it is fine.